### PR TITLE
CVE-2015-1267 and CVE 2010-4482

### PR DESCRIPTION
**Pull request by ericethegreat**
CVE-2010-4482 and CVE-2010-4482

### CODE REVIEW

File: cves/CVE-2015-1267.yml [hunk 1]
```diff
@@ -15,11 +15,13 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description: 
-bounty:
-  date: 
-  amount: 
-  references: []
+description: |
+    Scripts can pass any objects to the API as the chief context, meaning cross-origin scripting is possible. Therefore, information from one system can be passed into another and be considered as another system.
```
Comment (andymeneely) [inline cves/CVE-2015-1267.yml]: I could use a little more description of what the problem was. It's not clear that it's a javascript vulnerability until I saw v8. What APIs? Explain a little more about cross-origin scripting being possible.

File: cves/CVE-2010-4482.yml [hunk 2]
```diff
@@ -15,11 +15,12 @@ description_instructions: |
...
   keep too.
-description: 
+description: |
+  The vulnerability is such that Chrome is able to open a webpage as a pop-up, which can be abused as malicious users will generate unwanted pages by bypassing the pop-up blocker using JavaScript. Chrome did not check whether the input was generated by an actual user. It is basically an example of improper input validation.
 bounty:
-  date: 
-  amount: 
-  references: []
+  date: N/A
+  amount: 0.0
```
Comment (andymeneely) [inline cves/CVE-2010-4482.yml]: Leave this null - a bounty of zero is different than not giving out a bounty

File: cves/CVE-2010-4482.yml [hunk 3]
```diff
@@ -15,11 +15,12 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description: 
+description: |
+  The vulnerability is such that Chrome is able to open a webpage as a pop-up, which can be abused as malicious users will generate unwanted pages by bypassing the pop-up blocker using JavaScript. Chrome did not check whether the input was generated by an actual user. It is basically an example of improper input validation.
```
Comment (andymeneely) [inline cves/CVE-2010-4482.yml]: So is this requiring XSS to work in the first place? Or is it possible to exploit this another way?

File: cves/CVE-2010-4482.yml [hunk 4]
```diff
@@ -33,12 +34,28 @@ fixes_vcc_instructions: |
...
+    Added tests to ensure the action is user-initiated.
 - :commit: 12ea9b27f4b143c59b6cb749f729025e484e7c5d
-  :note: ''
-vccs: []
+  :note: |
+    Introduced a patch to test whenever a link is redirected to an unwanted pop-up.
+vccs:
+- :commit: 3a8eecb2bb859491b344be321abb4ff3d2b7f0f3
+  :note: | 
+    They were working on removing duplicates for process grouping logic.
```
Comment (andymeneely) [inline cves/CVE-2010-4482.yml]: Duplicates of what?

File: cves/CVE-2010-4482.yml [hunk 5]
```diff
@@ -89,16 +110,16 @@ subsystem:
...
+  name: v8
 interesting_commits:
   question: |
     Are there any interesting commits between your VCC(s) and fix(es)?
 
     Write a brief (under 100 words) description of why you think this commit was
     interesting in light of the lessons learned from this vulnerability. Any
     emerging themes?
-  commits:
+  commits: There were no interesting commits.
```
Comment (andymeneely) [inline cves/CVE-2010-4482.yml]: I'd like you to either (a) find some interesting commits to talk about, or (b) explain why NONE of the commits between the VCC and vulnerability were relevant. That way I know you actually did this part ;)

File: cves/CVE-2010-4482.yml [hunk 6]
```diff
@@ -1,25 +1,23 @@
...
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description: 
+description: |
+  The vulnerability is such that Chrome is able to open a webpage as a pop-up, which can be abused as malicious users will generate unwanted pages by bypassing the pop-up blocker using cross site-scripting (and by extension, cross-origin scripting). Chrome did not check whether the input was generated by an actual user. It is basically an example of improper input validation.
 bounty:
-  date: 
-  amount: 
-  references: []
+  date: N/A
```
Comment (xdmds) [inline cves/CVE-2010-4482.yml]: needs a date or else it won't show on the website. If it wasn't actually a bounty then you can omit entering it as one.

File: cves/CVE-2015-1267.yml [hunk 7]
```diff
@@ -15,11 +15,12 @@ description_instructions: |
...
   keep too.
-description: 
-bounty:
-  date: 
-  amount: 
-  references: []
+description: |
+  Scripts can pass any objects to the API as the chief context, meaning cross-origin scripting is possible. Therefore, information from one system can be passed into another and be considered as another system.   
+bounty: 
+  date: Unknown
```
Comment (xdmds) [inline cves/CVE-2015-1267.yml]: was it actually a bounty if there was no date and amount?

File: cves/CVE-2010-4482.yml [hunk 8]
```diff
@@ -77,10 +94,11 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then you may
     leave this part blank.
-  answer: 
-  date: 
-  automated: 
-  google: 
+  answer: |
+    The vulnerability was discovered by inserting an unwanted URL in a JavaScript opening the window and then calling the JavaScript through the URL of another website.
```
Comment (ijlondon) [inline cves/CVE-2010-4482.yml]: missing two commas "..JavaScript, opening.." & "..window, and.."

File: cves/CVE-2015-1267.yml [hunk 9]
```diff
@@ -60,9 +70,11 @@ unit_tested:
 
     For the "fix" answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
-  answer: 
-  code: 
-  fix: 
+  answer: |
+    The initial code did not involve any tests for unsafe context because the developers wrote the program to be used internally and did not realize that inputs from the public API could be introduced.
+    The fix involved automated tests that were later added when realizing that context could be unsafe from the public API. The final commit message added assert statements so that context would be checked.
```
Comment (ijlondon) [inline cves/CVE-2015-1267.yml]: "after realizing" or "when they realized that" tense correction

**Comment by ijlondon** [comment]
Delete the .DS_Store file

**Pull request merged**